pkg/kv/kvserver/replica_command.go

```diff
@@ -2451,6 +2451,10 @@ func (r *Replica) sendSnapshot(
 		Term:  snap.RaftSnap.Metadata.Term,
 	}
 
+	// See comment on DeprecatedUsingAppliedStateKey for why we need to set this
+	// explicitly for snapshots going out to followers.
+	snap.State.DeprecatedUsingAppliedStateKey = true
```

Comment:
I think if you did this in `stateloader.Load` you wouldn't have to think about this anywhere else.

Reply:
Yea, but I wanted this code to be close to the follower snapshot business.